src/ui.rs

```diff
@@ -1317,6 +1341,7 @@ where
 
             if r.contains_pointer() {
                 if snarl_state.has_new_wires_in() {
+                    // TODO: And here I am also not sure what these modifiers do
```

Comment:
When dragging new wires on the input side user may add new wire connected to another input pin by hovering over it with SHIFT and not COMMAND pressed.
Or remove one by hovering with COMMAND and not SHIFT pressed.